testing/sdk_cmd.py

```diff
@@ -22,6 +23,7 @@ def fn():
         return fn()
 
 
+@retry(stop_max_attempt_number=5, stop_max_delay=60)
```

Comment:
Even though the other invocations of this don't seem to be causing any harm since the tests are passing, it's not safe enough to just assume it'll be fine without proper confirmation - removing for now as well. If the flakes pop up again, we'll revisit.